src/main/java/gov/cancer/pageobject/crosscutting/BreadCrumbPage.java

```diff
@@ -1,25 +1,30 @@
 package gov.cancer.pageobject.crosscutting;
 
+import java.util.ArrayList;
+import java.util.List;
+
 import org.openqa.selenium.WebElement;
 import org.openqa.selenium.support.FindBy;
 import org.openqa.selenium.support.How;
 
+import gov.cancer.framework.ElementHelper;
 import gov.cancer.pageobject.PageObjectBase;
+import gov.cancer.pageobject.helper.BreadCrumb;
 
 /**
  * Pseudo page object representing any page in the system. The BreadCrumb class
  * is used solely for verifying attributes of a page's bread crumb trail.
  */
 public class BreadCrumbPage extends PageObjectBase {
 
-
-  @FindBy(how = How.CSS, using = ".breadcrumbs")
+  @FindBy(how = How.CSS, using = "div#cgvSlBreadcrumb")
   WebElement breadCrumb;
```

Comment:
Please do not use the `@FindBy` annotation. Use the `ElementHelper` class instead.